### PR TITLE
FSharp Bootcamp Unit 1

### PR DESCRIPTION
**Pull request by bobbychopra**
Added missing Akka.NET bootcamp docs for FSharp Unit 1

### CODE REVIEW

**Comment by Aaronontheweb** [comment]
Reviewing this now!

File: src/Unit-1/FSharp/lesson3/README.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+
+As you can see, we're now handing off the input from the console to the `validationActor` for validation and decisions. `consoleReaderActor` is now only responsible for reading from the console and handing the data off to another more sophisticated actor.
+
+There. That's better.
+
+### Once you're done
+Compare your code to the solution in the [Completed](Completed/) folder to see what the instructors included in their samples.
+
+If everything is working as it should, the output you see should be identical to last time:
+![Petabridge Akka.NET Bootcamp Lesson 1.2 Correct Output](Images/working_lesson3.jpg)
```
Comment (Aaronontheweb) [inline src/Unit-1/FSharp/lesson3/README.md]: This image is broken

File: src/Unit-1/FSharp/lesson4/README.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ Since parents supervise their children, this means that ***every actor has a sup
 Within your actor system, actors are arranged into a hierarchy. This means there are "top level" actors, which essentially report directly to the `ActorSystem` itself, and there are "child" actors, which report to other actors.
 
 The overall hierarchy looks like this (we'll go through piece by piece in a moment):
-![Petabridge Akka.NET Bootcamp Lesson 1.3 Actor Hierarchies](Images/hierarchy_overview.png)
+![Petabridge Akka.NET Bootcamp Lesson 1.3 Actor Hierarchies](Images//hierarchy_overview.png)
```
Comment (Aaronontheweb) [inline src/Unit-1/FSharp/lesson4/README.md]: image is broken

**Comment by Aaronontheweb** [comment]
Rather than comment on each instance individually - all of the images in 1.4 appear to be broken.

**Comment by bobbychopra** [comment]
Using Git Shell (Powershell based), renaming folder didn't identify/push local changes. On my machine, folder is "Images" (renamed it) but on github it is "images." Will figure out a way to resolve the issue

File: src/Unit-1/FSharp/lesson3/README.md (hunk 1 above)
Comment (skotzko) [inline src/Unit-1/FSharp/lesson3/README.md]: @bobbychopra this image is broken, otherwise looks awesome!

**Comment by skotzko** [comment]
@bobbychopra this looks awesome, thank you! merging.

**Pull request merged**